swagger-ui/dist/zipskee-v1.swagger.yaml

```diff
@@ -136,6 +136,8 @@ paths:
                type: string
              password:
                type: string
+             passwordConfirmation:
```

Comment:
@bad6e - do we need to pass the password confirmation to the server or could this be done client side?
cc: @TaeKimJR